src/index.html

```diff
@@ -4,9 +4,323 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo"><img src="./images/logo.png" alt="logo"></a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#apple" class="header__nav-link is-active">Apple</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#samsung" class="header__nav-link">Samsung</a>
+          </li>
```

Comment:
it's better for the readable
```suggestion
      <a href="#home" class="header__logo">
        <img src="./images/logo.png" alt="logo">
      </a>
      
      <nav class="header__nav">
        <ul class="header__nav-list">
          <li class="header__nav-item">
            <a href="#apple" class="header__nav-link is-active">Apple</a>
          </li>
          
          <li class="header__nav-item">
            <a href="#samsung" class="header__nav-link">Samsung</a>
          </li>
          
```